packages/react/src/components/MultiSelect/FilterableMultiSelect.tsx

```diff
@@ -380,8 +558,8 @@ const FilterableMultiSelect = React.forwardRef(function FilterableMultiSelect(
                 ) : null}
                 <ListBox
                   aria-label={deprecatedAriaLabel || ariaLabel}
-                  onFocus={isFluid ? handleFocus : null}
-                  onBlur={isFluid ? handleFocus : null}
+                  onFocus={isFluid ? handleFocus : undefined}
+                  onBlur={isFluid ? handleFocus : undefined}
```

Comment:
This one generally encountered when converting to TS since it's pretty usual to specify optional prop as `prop?: type` and that doesn't cover null's… [some might say that is a good thing](https://www.infoq.com/presentations/Null-References-The-Billion-Dollar-Mistake-Tony-Hoare/) and I tend to agree just based on simplicity on the definition side

Reply:
Yep I agree this is fine